sepolicy/vendor/file_contexts

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
merge issue?

Reply:
yes, thank you for pointing it out...  i pushed the wrong branch...